py_gtfs_rt_ingestion/poetry.lock

```diff
@@ -53,6 +53,14 @@ category = "dev"
 optional = false
 python-versions = ">=3.5"
 
+[[package]]
```

Comment:
this file is going to get update each time we update the `pyproject.toml` file. i think best practices say to keep it commited so that we know we're on the same version of everything.